docs/fuse-box/Quick-Start.mdx

```diff
@@ -9,91 +9,353 @@ Welcome to the Fuse Blockchain Developer Quick Start Guide! This guide is design
+  );
+}
+```
+
+### Step 5: Create a Smart Contract Wallet.
+To use the SDK in your project, you must create an instance of the FuseSDK class. This class is the key to interacting with the Fuse API using the SDK.
+
+The `init` method sets up a wallet and authenticates the user's provided credentials. It requires a API_KEY from FuseBox. Create an Account and get the `API_KEY` in the Fuse Developer Console [here](https://console.fuse.io). When you send a request to the server using this method, it verifies the credentials. If the credentials are valid, developers can use the `getSender()` to return an address.
+
+To create the Smart Contract Wallet, we will update the code by adding a `smartWallet` function, and parsing the returned Smart Contract Address to a state variable.
+
+Inside the Home Component, import the useState method:
+
+```javascript
+import { useState, useEffect } from 'react'
+```
+
+Set the state variable to manage the state of the Smart Contract Account that will be created:
+
+```javascript
+const [smartAccount, setSmartAccount] = useState<string>("")
+```
+
+Add the smartWallet function:
+
+```javascript
+const smartWallet = async() => {
+  const apiKey = "API_KEY";
+  const credentials = new ethers.Wallet(`0xPrivateKey`);
+  const fuseSDK = await FuseSDK.init(apiKey, credentials, {
+    withPaymaster: true,
+  });
+  setSmartAccount(fuseSDK.wallet.getSender());
+  console.log(
+      `Smart account address: https://explorer.fuse.io/address/${fuseSDK.wallet.getSender()}`
+  );
+ }
+```
+
+To call the function and render the response in the application UI, we have to use the `useEffect` hook to prevent too many re-renders:
+
+```javascript
+import { useState, useEffect } from ’react’;
+```
+
+Add the useEffect Hook:
+
+```javascript
+useEffect(()=> {
+  smartWallet()
+}, [])
+```
+
+Reload the app and open the console. The newly created Smart Contract Address will be logged into the Console!
+
+To return the Smart Contract Address to the UI, you can parse the stored state variable:
+
+```javascript
+ return (
+    <main className={`flex flex-col items-center p-24 ${inter.className}`}>
+      Hello, World!
+      {smartAccount}
+    </main>
+  );
+```
+
+### Step 6: Token Transfers
+Transfer Transactions which happen on the Blockchain are state-changing. In Account Abstraction these “Transactions” are called “UserOperations” because they are first sent to an alternate Mempool where they are then Bundled and sent on-chain. FuseBox uses the transferToken method to carry out a UserOperation of transferring a Token from one Address to another.
+
+Add another Function transferToken:
+
+```javascript
+const transferToken = async() => {
+  const tokenAddress = "0x28C3d1cD466Ba22f6cae51b1a4692a831696391A"
+  const apiKey = "API_KEY";
+  const credentials = new ethers.Wallet(`0xPrivateKey`);
+  const fuseSDK = await FuseSDK.init(apiKey, credentials, {
+    withPaymaster: true,
+  });
+  const recipient = "Enter ANY EOA or SCA address";
```

Comment:
@emmaodia Let's change "SCA" to "SCW" (Smart contract wallet). WDYT?

Reply:
Yes, I agree. Done. @LiorAgnin